pkg/pipeline/lineage.go

```diff
@@ -241,11 +241,11 @@ func (p *LineageExtractor) addColumnToAsset(asset *Asset, colName string, upstre
 	}
 
 	newCol := Column{
-		Name:       colName,
-		PrimaryKey: false,
-		Type:       upstreamCol.Type,
-		Checks:     []ColumnCheck{},
-		// Description:     upstreamCol.Description,
+		Name:            colName,
+		PrimaryKey:      false,
+		Type:            upstreamCol.Type,
+		Checks:          []ColumnCheck{},
+		Description:     upstreamCol.Description,
```

Comment:
would this override if there's an existing description for that column? we wouldn't want that to happen. on the other hand, if there's a column matching with the same name that has no description, we'd like the description to come from the upstream column.

can you please add some tests to ensure this is exactly what happens?